server/converters/cxgtool.py

```diff
@@ -409,6 +442,17 @@ def save_metadata(container, metadata):
     with tiledb.DenseArray(a_name, mode="w") as A:
         A.meta["cxg_version"] = CXG_VERSION
         A.meta["cxg_properties"] = json.dumps(metadata)
+        if extract_colors:
+            try:
+                A.meta["cxg_category_colors"] = json.dumps(
+                    convert_anndata_category_colors_to_cxg_category_colors(adata)
+                )
+            except (KeyError, ColorFormatException):
+                log(
+                    0,
+                    "Warning: failed to extract colors from h5ad file! "
+                    "Fix the h5ad file or rerun with --ignore-colors. See help for details.",
```

Comment:
this error message seems out of date, as there is no `--ignore-colors` option that I can see.

Reply:
True. I've thought about your comment below and decided to add this back and add a `--disable-user-colors` flag to `cellxgene launch`. This makes the behavior between cxgtool.py and the CLI consistent.